Kernel/Devices/PCISerialDevice.h

```diff
@@ -30,8 +30,9 @@ class PCISerialDevice {
         SerialDevice::Baud baud_rate { SerialDevice::Baud::Baud38400 };
     };
```

Comment:
Commit title should be "Kernel: Add PCISerialDevice WCH CH351 ids"

Also, did you actually test them? if that's the case, please add a few words explaining what has been done.

Reply:
I have updated the commit and PR description. On Discord in the bare-metal channel I have also shared a link to the PCI card that was used.